sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/channel/ManagedChannelFactory.java

```diff
@@ -17,7 +17,6 @@
  */
 package org.apache.beam.sdk.fn.channel;
 
-import avro.shaded.com.google.common.collect.ImmutableList;
```

Comment:
shoot!

Don't we have checkstyle checks for this?

Reply:
We do, it also failed.